lib/MTT/MPI/Get/OMPI_Snapshot.pm

```diff
@@ -13,7 +13,7 @@
 
 package MTT::MPI::Get::OMPI_Snapshot;
 
-use JSON;
+use JSON::JSON;
```

Comment:
I think this is ok since it's referencing the one in `lib` so we look in the `JSON` directory then the `JSON.pm`. I say merge it and see what happens.